core/src/test/scala/org/apache/spark/sql/types/pk/MultiColumnPKDataTypeSuites.scala

```diff
@@ -59,6 +59,7 @@ trait MultiColumnPKDataTypeSuites extends BaseRandomDataTypeTest {
     val size = dataTypes.size - 1
     dataTypes.indices
     // filter i<j rather than i!=j to exclude the same tuple (ignore order)
+    // We test if we can fetch the correct data, I think the order of col in pk won't affect it
```

Comment:
It's under the assumption that the order of col in pk wouldn't affect the result of the test.